README.md

```diff
@@ -1,11 +1,42 @@
-## Medical-Image-Registration-Short-Course
+<div style="text-align: center;" align="center">
+  <h1> :medical_symbol:  Medical-Image-Registration-Short-Course </h1>
+  <h2> <a href="https://healthbioscienceideas.github.io/Medical-Image-Registration-Short-Course/">:page_with_curl: Sandpaper</a> </h2>
+</div>
 
-   This is the lesson repository for [Medical-Image-Registration-Short-Course](https://healthbioscienceideas.github.io/Medical-Image-Registration-Short-Course)
-
-## Setting up and preview sandpaper 
+## :nut_and_bolt: Setting up and preview sandpaper 
 See [details](_setting_up) to setup sandpaper and its dependencies. Otherwise you are ready to preview your work:
 ```
 R
 sandpaper::build_lesson()
 ```
 
+## 🤝 Contributing
+Medical-Image-Registration-Short-Course follows the [Contributor Covenant Code of Conduct](.github/CODE_OF_CONDUCT.md).
+Contributions, issues and feature requests are welcome, so feel free to open new [issues](https://github.com/HealthBioscienceIDEAS/Medical-Image-Registration-Short-Course/issues/new/choose).
+We also suggest to checking [the contributing guide](CONTRIBUTING.md).
+
+## :octocat: Cloning repository
+* Generate your SSH keys as suggested [here](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
+* Clone the repository by typing (or copying) the following lines in a terminal
+```
+git clone git@github.com:HealthBioscienceIDEAS/Medical-Image-Registration-Short-Course.git
+```
+
+## Maintainers
+* [Dave Cash](https://github.com/davecash75)
+* [Jamie Mcclelland](https://github.com/jamie-mcclelland)
+
+## Contributors
+This project exists thanks to all the people who contribute.
+* [Milan Malfait](https://github.com/milanmlft)
+* [Zakaria Senousy](https://github.com/zsenousy)
+* [Stephen Thompson](https://github.com/thompson318)
+* [Miguel Xochicale](https://github.com/mxochicale)
```

Comment:
Can you add Dave and Jamie?

Reply:
Already there as maintainers https://github.com/HealthBioscienceIDEAS/Medical-Image-Registration-Short-Course/tree/15-reproducible-repo?tab=readme-ov-file#maintainers